python/tvm/relax/dpl/__init__.py

```diff
@@ -19,3 +19,4 @@
 
 from .pattern import *
 from .context import *
+from .rewrite import rewrite_call, rewrite_bindings
```

Comment:
The existing `rewrite` function is renamed to `rewrite_call` to make it clear that it is CallNode rewriting. And together with the new dataflow block rewriting function, it is put under the new file.